apps/backend/package-lock.json

```diff
@@ -79,30 +78,38 @@
         "@types/sinon": "^9.0.11",
         "@types/validator": "^13.7.0",
         "@types/yup": "^0.29.13",
-        "@typescript-eslint/eslint-plugin": "^5.38.0",
-        "@typescript-eslint/parser": "^5.38.0",
+        "@typescript-eslint/eslint-plugin": "^7.7.0",
```

Comment:
This is tageting node 18.18 or higher but our Dockerfile.dev is using node 18.17 could you update it?